packages/create-redwood-app/src/create-redwood-app.js

```diff
@@ -57,6 +57,11 @@ if (!targetDir) {
 }
 
 const newAppDir = path.resolve(process.cwd(), targetDir)
+const appDirExists = fs.existsSync(newAppDir)
+
+if (appDirExists && fs.readdirSync(newAppDir).length > 0) {
+  throw new Error(`'${newAppDir}' already exists and is not empty.`)
```

Comment:
This would result in an uncaught exception, it would be better to use a combination of `console.log` and `process.exit(1)`